### ISSUE TITLE
Installing templates from repository does not install all templates

### ISSUE THREAD

**Issue by damianesteban** [fermyon/spin#679]
I was taking a look at this tutorial [here](https://www.fermyon.com/blog/spinning-with-swift) and I ran:

```
spin templates install --git https://github.com/fermyon/spin
```
then:
```
spin new http-swift hello-there
```
and received the following error:
```
Template http-swift not found
```

I see it in the repo, is there a way I manually install it?

**Comment by fibonacci1729** [comment]
@damianesteban Are you running a released version (and if so what version) or did you build from main?

**Comment by damianesteban** [comment]
Version:

```
spin 0.4.0 (784094e 2022-07-13)
```

**Comment by fibonacci1729** [comment]
@damianesteban I believe if you do `spin templates install --branch main --git https://github.com/fermyon/spin` you will pick up the newest templates included after `0.4.0`.

**Comment by damianesteban** [comment]
Yes that works. Thank you.
